CMakeLists.txt

```diff
@@ -41,21 +41,28 @@ endif()
 defoption(LEATHERMAN_MOCK_CURL "Use mock curl library for testing Leatherman.curl" ${MOCK_CURL})
 
 add_definitions(${LEATHERMAN_DEFINITIONS})
-add_definitions(-DLEATHERMAN_LOCALE_INSTALL="${CMAKE_INSTALL_PREFIX}/share/locale")
+if (LEATHERMAN_LOCALE_VAR AND LEATHERMAN_LOCALE_INSTALL)
+    # Add an environment variable to look up install prefix at runtime.
+    add_definitions(-DLEATHERMAN_LOCALE_VAR="${LEATHERMAN_LOCALE_VAR}"
+                    -DLEATHERMAN_LOCALE_INSTALL="${LEATHERMAN_LOCALE_INSTALL}")
+else()
+    # Add install location instead.
+    add_definitions(-DLEATHERMAN_LOCALE_INSTALL="${CMAKE_INSTALL_PREFIX}/share/locale")
```

Comment:
So if we don't set these environment vars on Unix (was looking at the linked client-tools PR), it will look in `/usr/local/share/locale` for translation files? I'm pretty sure at least for puppet-agent that is not what we want, so I assume we will need to set these for Leatherman there as well, probably on all platforms, to the paths stated [here](https://github.com/puppetlabs/puppet-specifications/blob/master/file_paths.md)?